ghost/core/core/frontend/helpers/comment_count.js

```diff
@@ -3,23 +3,24 @@ const {labs} = require('../services/proxy');
 
 function commentCount(options) {
     return new SafeString(`
-       <span data-ghost-comment-count="${this.id}" style="display:none">
-           ${options.fn(this)}
-       </span>`
+       <script
```

Comment:
Should this perhaps be a visible element like a `span` so that we can style it whilst comments are loading?

Reply:
I think that could be handy

Reply:
As discussed in notion - we're going to leave this for now